pyomo/core/expr/numvalue.py

```diff
@@ -99,12 +99,6 @@ def __setstate__(self, state):
 native_boolean_types = set([ int, bool, str ])
 native_logical_types = {bool, }
 pyomo_constant_types = set()  # includes NumericConstant
-try:
-    native_numeric_types.add(long)
-    native_integer_types.add(long)
-    native_boolean_types.add(long)
-except:
-    pass
```

Comment:
Why is this no longer needed?

Reply:
`long` no longer exists in Python 3.x. It's a Python 2 integer type.